src/Pawn.cpp

```diff
@@ -0,0 +1,29 @@
+#include "..\include\Pawn.h"
+
+Pawn::Pawn(bool isWhite): Piece(isWhite)
+{
+}
+
+bool Pawn::isAcceptableMove(const Position& source, const Position& dest) const
+{
+    if (m_thisIsFirstMove)
+        if (this->getPieceColor())
+            return (dest.x == source.x + 2 || dest.x == source.x + 1) && dest.y == source.y;

```

Comment:
It is generally recommended to add braces {} for nested if statements to improve code readability and reduce the chances of introducing bugs.